README.md

```diff
@@ -76,6 +76,8 @@ To install ckanext-multilang:
 8. Restart CKAN. For example if you've deployed CKAN with Apache on Ubuntu:
 
      sudo service apache2 reload
+     
+>**_TIP:_** You can also enable/disable tag localization using the `multilang.enable_tag_localization` (with default `False`) settings key (in `prodcution.ini`) by settings it to boolean `True/False`.
```

Comment:
Not a tip, it's a configuration option worth of more attention from the installer. Please move this as point 6 (along with the other points about editing the ini file. Also correct the typo ("prodcution").